### DISCUSSION TITLE
Question about scraping events

### DISCUSSION THREAD

**Discussion by VladyslavDoc** [rustdesk/rustdesk#7120] [Q&A]
Hello!
First of all, thank you for your work. It's amazing.
I have question, may be you can help me. Could I get coordinates of event and screencap during connection? We can imagene that I want to record some action and save information about action.

**Comment by rustdesk** [comment]
Our self-hosting Pro does support event log. 
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/#:~:text=Log%20management%20(Connection%2C%20file%20transfer%2C%20alarm%2C%20etc.)

**Reply by VladyslavDoc** [reply]
Am I correct that with the Pro version I can set up a pipeline where during a certain action such as a touch I take a screenshot and save the coordinates of the touch?
